homeassistant/components/homematic/binary_sensor.py

```diff
@@ -1,35 +1,31 @@
 """Support for HomeMatic binary sensors."""
 from homeassistant.components.binary_sensor import (
-    DEVICE_CLASS_BATTERY,
-    DEVICE_CLASS_MOTION,
-    DEVICE_CLASS_OPENING,
-    DEVICE_CLASS_PRESENCE,
-    DEVICE_CLASS_SMOKE,
+    BinarySensorDeviceClass,
     BinarySensorEntity,
 )
 
 from .const import ATTR_DISCOVER_DEVICES, ATTR_DISCOVERY_TYPE, DISCOVER_BATTERY
 from .entity import HMDevice
 
 SENSOR_TYPES_CLASS = {
-    "IPShutterContact": DEVICE_CLASS_OPENING,
-    "IPShutterContactSabotage": DEVICE_CLASS_OPENING,
-    "MaxShutterContact": DEVICE_CLASS_OPENING,
-    "Motion": DEVICE_CLASS_MOTION,
-    "MotionV2": DEVICE_CLASS_MOTION,
-    "PresenceIP": DEVICE_CLASS_PRESENCE,
+    "IPShutterContact": BinarySensorDeviceClass.OPENING,
+    "IPShutterContactSabotage": BinarySensorDeviceClass.OPENING,
+    "MaxShutterContact": BinarySensorDeviceClass.OPENING,
+    "Motion": BinarySensorDeviceClass.MOTION,
+    "MotionV2": BinarySensorDeviceClass.MOTION,
+    "PresenceIP": BinarySensorDeviceClass.PRESENCE,
```

Comment:
```suggestion
    "PresenceIP": BinarySensorDeviceClass.MOTION,
```
You did it right, but that is a old bug and a good time to fix that